sdks/go/examples/snippets/01_03intro.go

```diff
@@ -0,0 +1,93 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package snippets
+
+import (
+	"flag"
+
+	"github.com/apache/beam/sdks/go/pkg/beam"
+	"github.com/apache/beam/sdks/go/pkg/beam/io/textio"
+)
+
+// PipelineConstruction contains snippets for the initial sections of
+// the Beam Programming Guide, from initializing to submitting a
+// pipeline.
+func PipelineConstruction() {
+	// [START pipeline_options]
+	// If beamx or Go flags are used, flags must be parsed first,
+	// before beam.Init() is called.
+	flag.Parse()
+	// [END pipeline_options]
+
+	// [START pipelines_constructing_creating]
+	// beam.Init() is an initialization hook that must be called near
+	// the beginging of main().
+	beam.Init()
+
+	// Create the Pipeline object and root scope.
+	pipeline, scope := beam.NewPipelineWithRoot()
+	// [END pipelines_constructing_creating]
+
+	// [START pipelines_constructing_reading]
+	lines := textio.Read(scope, "gs://some/inputData.txt")
```

Comment:
Suggested comment insertion:
```
// Read the file at the URI 'gs://some/inputData.txt' and return the lines as a PCollection<string>.  
// Notice the scope as the first variable when calling the method as is needed when calling all transforms.
lines := textio.Read(scope, "gs://some/inputData.txt")
```

Reply:
Done. Agreed that this is probably the best, first place to mention what the scope variable is for in code.

I've added a similar addendum to the Overview talking about the `Scope` variable explicitly, and that we abbreviate it to `s` conventionally. At that point I use the convention.